src/components/MultiSelect/MultiSelectDialog.tsx

```diff
@@ -0,0 +1,222 @@
+import React, { useRef, useState, forwardRef } from "react";
+import { chakra } from "@chakra-ui/react";
+import {
+  Box,
+  ListItem,
+  UnorderedList,
+  useMultiStyleConfig,
+  useOutsideClick,
+  useMergeRefs,
+} from "@chakra-ui/react";
+// @TODO Add "@chakra-ui/focus-lock" to package.json dependencies ?
+import FocusLock from "@chakra-ui/focus-lock";
+
+import Button from "./../Button/Button";
+import ButtonGroup from "./../ButtonGroup/ButtonGroup";
+import Checkbox from "./../Checkbox/Checkbox";
+import { MultiSelectItem, MultiSelectProps } from "./MultiSelect";
+import MultiSelectMenuButton from "./MultiSelectMenuButton";
+import useNYPLBreakpoints from "./../../hooks/useNYPLBreakpoints";
+
+type MultiSelectDialogProps = Omit<MultiSelectProps, "onChange"> & {
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+};
+
+// Forward ref collides with the ref created within this Component for registring
+// ouside element clicks to close the list
+// noticed closeing is not working reliably anyways...?
```

Comment:
You can remove this. Clicking outside the element to close does work if you are clicking within the component's iframe that  Storybook provides. It only appears as if it is not working because the iframe is small compared to the entire Storybook page.

Reply:
Removed.